pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -803,6 +803,38 @@ var _ = Describe("Converter", func() {
 			Expect(domain.Spec.Devices.Interfaces[0].BootOrder.Order).To(Equal(uint(bootOrder)))
 			Expect(domain.Spec.Devices.Interfaces[1].BootOrder).To(BeNil())
 		})
+		table.DescribeTable("should allow setting driver", func(drivers []string) {
+			for i, driver := range drivers {
+				name := fmt.Sprintf("Name%d", i)
+				iface := v1.DefaultNetworkInterface()
+				net := v1.DefaultPodNetwork()
+				iface.Name = name
+				iface.Driver = driver
+				net.Name = name
+				vmi.Spec.Domain.Devices.Interfaces = append(vmi.Spec.Domain.Devices.Interfaces, *iface)
+				vmi.Spec.Networks = append(vmi.Spec.Networks, *net)
+			}
+			domain := vmiToDomain(vmi, c)
+
+			Expect(len(domain.Spec.Devices.Interfaces)).To(Equal(len(drivers)))
+			for i, driver := range drivers {
+				if driver == "" {
+					// If empty, then the domain defaults to 'vhost' if vhost device is present,
+					// but silently falls back to 'qemu' without error. (Since 0.8.8)
+					Expect(domain.Spec.Devices.Interfaces[i].Driver).To(BeNil())
+				} else {
+					Expect(domain.Spec.Devices.Interfaces[i].Driver.Name).To(Equal(driver))
+				}
+			}
+		},
+			table.Entry("one interface backed by default driver", []string{""}),
```

Comment:
Would it make sense for us to set the default explicitly? Just for transparency and just a nit.

Reply:
Seems not to me. With introducing const driver names in api side, I will remove these. If I misunderstood your point, please correct me later in a patch.